Neos.Neos/Documentation/References/NeosFusionReference.rst

```diff
@@ -1001,6 +1010,149 @@ found, it is used.
 Only if the current node is not available at all (even after considering default presets with their fallback rules),
 no node be assigned (so no link will be created and the items will have the ``absent`` state.)
 
+.. _Neos_Neos__MenuItems:
+
+Neos.Neos:MenuItems
+-------------------
+
+Create a list of menu-items items for nodes.
+
+:entryLevel: (integer) Start the menu at the given depth
+:maximumLevels: (integer) Restrict the maximum depth of items in the menu (relative to ``entryLevel``)
+:startingPoint: (Node) The parent node of the first menu level (defaults to ``node`` context variable)
+:lastLevel: (integer) Restrict the menu depth by node depth (relative to site node)
+:filter: (string) Filter items by node type (e.g. ``'!My.Site:News,Neos.Neos:Document'``), defaults to ``'Neos.Neos:Document'``
+:renderHiddenInIndex: (boolean) Whether nodes with ``hiddenInIndex`` should be rendered, defaults to ``false``
+:itemCollection: (array) Explicitly set the Node items for the menu (alternative to ``startingPoints`` and levels)
+
+MenuItems item properties:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:node: (Node) A node instance (with resolved shortcuts) that should be used to link to the item
+:originalNode: (Node) Original node for the item
+:state: (string) Menu state of the item: ``'normal'``, ``'current'`` (the current node) or ``'active'`` (ancestor of current node)
+:label: (string) Full label of the node
+:menuLevel: (integer) Men^u level the item is rendered on
+
+Examples:
+^^^^^^^^^
+
+::
+
+	menuItems = Neos.Neos:MenuItems {
+		entryLevel = 1
+		maximumLevels = 3
+	}
+
+MenuItems including site node:
+""""""""""""""""""""""""""""""
+
+::
+
+	menuItems = Neos.Neos:MenuItems {
+		itemCollection = ${q(site).add(q(site).children('[instanceof Neos.Neos:Document]')).get()}
+	}
+
+Menu with custom starting point:
+""""""""""""""""""""""""""""""""
+
+::
+
+	menuItems = Neos.Neos:MenuItems {
+		entryLevel = 2
+		maximumLevels = 1
+		startingPoint = ${q(site).children('[uriPathSegment="metamenu"]').get(0)}
```

Comment:
nitpick: IMO it's not a very good practice to refer to the `uriPathSegment` property of a node that should not even _have_ an uri path :)
```suggestion
		startingPoint = ${q(site).children('metamenu').get(0)}
```
Wouldn't that work if the metamenu node was auto-generated (as it should be)

Reply:
This is a 1:1 copy of the old Menu documentation. Maybe we should improve both in a second pr

Reply:
Ah - sure, just a tiny nitpick anyways